pyproject.toml

```diff
@@ -49,7 +49,7 @@ dependencies = [
   "anyascii>=0.1.5",
   "telepath>=0.3.1,<1",
   "laces>=0.1,<0.2",
-  "django-tasks>=0.8,<0.9",
+  "django-tasks>=0.8,<0.10",
```

Comment:
`0.9` makes a fair few changes, notably dropping `enqueue_on_commit`, which we currently have guidance on. Should this be bumped to `>=0.9`, and the guidance removed?

Reply:
Let's leave it at `>=0.8` for now, so that if anyone is using django-tasks more extensively in their projects in ways that would be impacted by the breaking changes in 0.9, they have a window in which to apply the necessary changes. We can consider bumping this in whichever release we start formally supporting Django 6.0.

(As far as I can see the only mention of `ENQUEUE_ON_COMMIT` is in the 6.4 release notes, and we'd probably want to keep it there for completeness along with a brief note to say that it no longer applies as of >=0.9. There's also a bunch of `captureOnCommitCallbacks` we can remove from the tests...)